pom.xml

```diff
@@ -12,16 +12,15 @@
 	</parent>
 	<properties>
 		<java.version>1.7</java.version>
-		<kafka.version>0.10.1.1</kafka.version>
-		<spring-kafka.version>1.1.2.RELEASE</spring-kafka.version>
-		<spring-integration-kafka.version>2.1.0.RELEASE</spring-integration-kafka.version>
+		<kafka.version>0.10.2.1</kafka.version>
+		<spring-kafka.version>1.2.0.RELEASE</spring-kafka.version>
```

Comment:
This a bit of an issue if it is a hard requirement, as applications created with initializr/Boot 1.5.4 will not get this dependency. Based on the fact that Spring Integration stays at 2.1.x I suppose that we can revert to 1.1.x ?